packages/contracts-watr/contracts/TokenControllerV3.sol

```diff
@@ -30,6 +30,12 @@ import {IProofOfReserveToken} from "./interface/IProofOfReserveToken.sol";
  * which can only be refilled by the owner.
 */
 
+interface HasOwner {
```

Comment:
I think this is handled better in #1245 : extracting the interface from ClaimableOwnable.